Lib/idlelib/idle_test/test_outwin.py

```diff
@@ -1,6 +1,7 @@
 "Test outwin, coverage 76%."
 
 from idlelib import outwin
+import sys
```

Comment:
Maybe it'd look nicer if you put it above the `from` bit.

Reply:
In test files, I put the import of the file being tested first.  Otherwise you would be right.